components/nexusoperations/frontend/fx.go

```diff
@@ -31,6 +32,16 @@ func ConfigProvider(coll *dynamicconfig.Collection) *Config {
 	}
 }
 
+func MetricTagConfigProvider(dc *dynamicconfig.Collection) *dynamicconfig.GlobalCachedTypedValue[*nexusoperations.NexusMetricTagConfig] {
+	return dynamicconfig.NewGlobalCachedTypedValue(
+		dc,
+		nexusoperations.MetricTagConfiguration,
+		func(config nexusoperations.NexusMetricTagConfig) (*nexusoperations.NexusMetricTagConfig, error) {
+			return &config, nil
+		},
+	)
+}
```

Comment:
Instead of having this a separate dependency, include another field in the `Config` object which already stores other dynamic configs.